src/main/java/hudson/plugins/ircbot/v2/PircListener.java

```diff
@@ -35,189 +40,299 @@
  */
 public class PircListener extends ListenerAdapter<PircBotX> {
 
-	private static final Logger LOGGER = Logger.getLogger(PircListener.class.getName());
-	
+	private static final Logger LOGGER = Logger.getLogger(PircListener.class
+			.getName());
```

Comment:
Please avoid whitespace-only changes (linebreak, changing indentation) like here and below.
They make it very difficult to spot the real changes